superset/views/schedules.py

```diff
@@ -83,6 +83,11 @@ class EmailScheduleView(
             description="List of recipients to send test email to. "
             "If empty, we send it to the original recipients",
         ),
+        "test_slack_channel": StringField(
+            "Test Slack Channel",
+            default=None,
+            description="A slack channel to send test message to.",
```

Comment:
`description="A slack channel to send a test message to.",`

Reply:
Also missing translations:

``` python
 from flask_babel import lazy_gettext as _

"test_slack_channel": StringField(
            _("Test Slack Channel"),
            default=None,
            description=_("A slack channel to send test message to."),
...
....

Reply:
this file doesn't have translations, should probably be a separate PR